hstream-diffflow/src/DiffFlow/Shard.hs

```diff
@@ -36,6 +36,7 @@ import           DiffFlow.Error
 import           DiffFlow.Graph
 import           DiffFlow.Types
 import qualified DiffFlow.Weird          as Weird
+import           HStream.Logger          (trace)
```

Comment:
Using qualified import is preferred. Since the base also has a `trace` function.